SQL/activity/history.sql

```diff
@@ -0,0 +1,7 @@
+SELECT
+    *
+FROM
+    activities
+WHERE
+    user_id = %s
```

Comment:
Why this?

Reply:
to avoid any leakage in the data! should I remove it?
I you prefer removin it, I should drop it from the table as well

Reply:
What do you think may happen if you remove it?

Reply:
If you prefer removin it, I should drop it from the table as well

Reply:
Also why drop? Not following.

Reply:
I am not sure what I was thinking! but I though maybe it's better to check the current user is the one querying!

Reply:
if we are not cross checking user_id, why we include it in the activity table as a foreign key?! :|